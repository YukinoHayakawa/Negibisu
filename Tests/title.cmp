Line 1, Col 1: [SHARP] #
Line 1, Col 2: [STRING_LITERAL] test
Line 1, Col 6: [COLON] :
Line 1, Col 7: [STRING_LITERAL] title
Line 1, Col 12: [NEWLINE]
Line 2, Col 2: [SHARP] #
Line 2, Col 4: [STRING_LITERAL] t est
Line 2, Col 10: [COLON] :
Line 2, Col 12: [STRING_LITERAL] tit le # t est
Line 2, Col 27: [COLON] :
Line 2, Col 29: [STRING_LITERAL] tit le
Line 2, Col 35: [NEWLINE]
Line 3, Col 1: [STRING_LITERAL] this is not a # title nor #this
Line 3, Col 32: [NEWLINE]
Line 4, Col 1: [NEWLINE]
Line 5, Col 1: [SHARP] #
Line 5, Col 2: [STRING_LITERAL] conutiunation
Line 7, Col 5: [COLON] :
Line 7, Col 6: [STRING_LITERAL] test title
