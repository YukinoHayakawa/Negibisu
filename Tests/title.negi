#test:title
 # t est : tit le # t est : tit le
this is not a # title nor #this

#\
 conutiunation\
    :\
        test ti\
tle\
