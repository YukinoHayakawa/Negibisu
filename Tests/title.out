Token Stream
============

Line 1, Col 1: [SHARP] #
Line 1, Col 2: [STRING_LITERAL] test
Line 1, Col 6: [COLON] :
Line 1, Col 7: [STRING_LITERAL] title
Line 1, Col 12: [NEWLINE] 
Line 2, Col 2: [SHARP] #
Line 2, Col 4: [STRING_LITERAL] t est
Line 2, Col 10: [COLON] :
Line 2, Col 12: [STRING_LITERAL] tit le # t est
Line 2, Col 27: [COLON] :
Line 2, Col 29: [STRING_LITERAL] tit le
Line 2, Col 35: [NEWLINE] 
Line 3, Col 1: [STRING_LITERAL] this is not a # title nor #this
Line 3, Col 32: [NEWLINE] 
Line 4, Col 1: [NEWLINE] 
Line 5, Col 1: [SHARP] #
Line 6, Col 2: [STRING_LITERAL] conutiunation
Line 7, Col 5: [COLON] :
Line 8, Col 9: [STRING_LITERAL] test title

AST (Parsed)
============

Error at Line 2, Col 27: Expected dialog text or a command.
SCRIPT
  SECTION: script_name="test", display_name="title"
  SECTION: script_name="t est", display_name="tit le # t est"
    NARRATOR_SAY: text="this is not a # title nor #this"
    SYSTEM_WAIT_INPUT
  SECTION: script_name="conutiunation", display_name="test title"

AST (Checked)
=============

SCRIPT
  SECTION: script_name="test", display_name="title"
  SECTION: script_name="t est", display_name="tit le # t est"
    NARRATOR_SAY: text="this is not a # title nor #this"
    SYSTEM_WAIT_INPUT
  SECTION: script_name="conutiunation", display_name="test title"

Symbol Tables: test
========================

test: Script

String Literals: test
========================


Target Code: test
========================


Symbol Tables: t est
========================

t est: Script

String Literals: t est
========================

"this is not a # title nor #this"

Target Code: t est
========================

narrator:say("this is not a # title nor #this");
_w();

Symbol Tables: conutiunation
========================

conutiunation: Script

String Literals: conutiunation
========================


Target Code: conutiunation
========================

